www/templates/html/Controller.tpl.php

```diff
@@ -1,27 +1,32 @@
 <?php
-require_once 'UNL/Templates.php';
+use UNL\Templates\Templates;
 
-UNL_Templates::$options['version'] = 4.0;
+$page = Templates::factory('Fixed', Templates::VERSION_4_1);
 
-$template = 'Fixed';
 $baseUrl = UNL_MediaHub_Controller::getURL();
 
-$page = UNL_Templates::factory($template);
-
-if (isset($GLOBALS['UNLTEMPLATEDEPENDENTSPATH'])) {
```

Comment:
What do you think of my solution here? https://github.com/unl/UNL_MediaHub/commit/b4ca9fdee6ddc1145cea5327efa8520d8d54484d